app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/HealthRecordPlaceholderFragment.kt

```diff
@@ -94,21 +95,21 @@ class HealthRecordPlaceholderFragment : Fragment(R.layout.health_record_placehol
     private fun collectHealthRecordsFlow() {
         viewLifecycleOwner.lifecycleScope.launch {
             viewLifecycleOwner.repeatOnLifecycle(Lifecycle.State.RESUMED) {
-                viewModel.patientHealthRecords.collect { records ->
+                viewModel.patients.collect { records ->
                     navigate(records)
                 }
             }
         }
     }
 
-    private fun navigate(records: List<PatientHealthRecord>) {
+    private fun navigate(records: List<PatientDto>) {
         if (records.isNotEmpty()) {
             if (records.size == 1) {
                 val action =
                     HealthRecordPlaceholderFragmentDirections
                         .actionHealthRecordsPlaceHolderFragmentToIndividualHealthRecordFragment(
-                            records.first().patientId,
-                            records.first().name
+                            records.first().id,
+                            records.first().fullName
                         )
                 findNavController().navigate(action)
```

Comment:
2022-04-22 22:01:06.487 3973-3973/ca.bc.gov.myhealth E/AndroidRuntime: FATAL EXCEPTION: main
    Process: ca.bc.gov.myhealth, PID: 3973
    java.lang.IllegalArgumentException: Navigation action/destination ca.bc.gov.myhealth:id/action_healthRecordsPlaceHolderFragment_to_individualHealthRecordFragment cannot be found from the current destination Destination(ca.bc.gov.myhealth:id/healthPassFragment) label=HealthPassFragment class=ca.bc.gov.bchealth.ui.healthpass.HealthPassFragment
        at androidx.navigation.NavController.navigate(NavController.java:940)
        at androidx.navigation.NavController.navigate(NavController.java:877)
        at androidx.navigation.NavController.navigate(NavController.java:863)
        at androidx.navigation.NavController.navigate(NavController.java:1159)
        at ca.bc.gov.bchealth.ui.healthrecord.HealthRecordPlaceholderFragment.navigate(HealthRecordPlaceholderFragment.kt:114)
        at ca.bc.gov.bchealth.ui.healthrecord.HealthRecordPlaceholderFragment.access$navigate(HealthRecordPlaceholderFragment.kt:25)
        at ca.bc.gov.bchealth.ui.healthrecord.HealthRecordPlaceholderFragment$collectHealthRecordsFlow$1$1$1.emit(HealthRecordPlaceholderFragment.kt:99)
        at ca.bc.gov.bchealth.ui.healthrecord.HealthRecordPlaceholderFragment$collectHealthRecordsFlow$1$1$1.emit(HealthRecordPlaceholderFragment.kt:98)
        at ca.bc.gov.bchealth.ui.healthrecord.HealthRecordPlaceholderViewModel$special$$inlined$map$1$2.emit(Emitters.kt:224)
        at ca.bc.gov.data.datasource.local.PatientLocalDataSource$special$$inlined$map$3$2.emit(Collect.kt:137)
        
        
        
       Got crash when I installed fresh app  > uploaded a QR for vaccine proof > Clicked in records tab